funsite/templates/lms/main.html

```diff
@@ -0,0 +1,204 @@
+    ## templates have access to these functions, and we can import these
+    ## into non-inheriting templates via the %namespace tag.
+    <%def name="theme_enabled()">
+  <% return settings.FEATURES.get("USE_CUSTOM_THEME", False) %>
+    </%def>
+    <%def name="is_microsite()">
+  <% return microsite.is_request_in_microsite() %>
+    </%def>
+
+    <%def name="stanford_theme_enabled()">
+  <%
+    if not theme_enabled():
+      return False
+
+    return getattr(settings, "THEME_NAME", None) == "stanford"
+  %>
+    </%def>
+
+    ## this needs to be here to prevent the title from mysteriously appearing in the body, in one case
+    <%def name="pagetitle()" />
+  <%block name="title">
+      <title>
+       ${page_title_breadcrumbs(self.pagetitle())}
+      </title>
+  </%block>
+
+  % if not allow_iframing:
+      <script type="text/javascript">
+        /* immediately break out of an iframe if coming from the marketing website */
+        (function (window) {
+            if (window.location !== window.top.location) {
+                window.top.location = window.location;
+            }
+        })(this);
+    </script>
+  % endif
+
+  <script type="text/javascript" src="/i18n.js"></script>
+
+  <link rel="icon" type="image/x-icon" href="${static.url(microsite.get_value('favicon_path', settings.FAVICON_PATH))}" />
+
+  <%static:css group='style-vendor'/>
+  ## We could do <%static:css group='style-main'/>, but that's only useful
+  ## if the group contains multiple files, and the 'style-main' group doesn't.
+  ## Instead, we'll construct this <link> element manually, to improve clarity.
+  ## When nothing in the system is referencing the 'style-main' group, it can
+  ## be removed from the environment file.
+  <%
+  application_css_path = "css/lms-main{rtl}.css".format(
+    rtl="-rtl" if get_language_bidi() else "",
+  )
+  %>
+    <link rel="stylesheet" href="${static.url(application_css_path)}" type="text/css" media="all" />
+
+    % if disable_courseware_js:
+    <%static:js group='base_vendor'/>
+    <%static:js group='base_application'/>
+  % else:
+    <%static:js group='main_vendor'/>
+    <%static:js group='application'/>
+  % endif
+
+  <script>
+    window.baseUrl = "${settings.STATIC_URL}";
+    (function (require) {
+        require.config({
+            baseUrl: window.baseUrl
+        });
+    }).call(this, require || RequireJS.require);
+</script>
+  <script type="text/javascript" src="${static.url("lms/js/require-config.js")}"></script>
+
+  % if not disable_courseware_js:
+    <%static:js group='module-js'/>
+  % endif
+
+  <%block name="headextra"/>
+
+  <%static:optional_include_mako file="head-extra.html" with_microsite="True" />
+
+  <%include file="widgets/optimizely.html" />
+  <%include file="widgets/segment-io.html" />
+
+  <meta name="path_prefix" content="${EDX_ROOT_URL}">
+
+  ## Disable google analytics for all inria courses
+  % if not "inria" in request.path_info:
```

Comment:
* Can we make it configurable `via` a setting?
* Instead of testing a single word in an URL (which is weak), can we handle a list of white/black listed root URLs?